packages/react/src/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,136 @@
+import * as RadixCheckbox from "@radix-ui/react-checkbox";
+import * as RadixLabel from "@radix-ui/react-label";
+import { useId } from "@reach/auto-id";
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import { Flex } from "../flex";
+import { Text } from "../text";
+import * as styles from "./Checkbox.css";
+
+type CheckboxProps = ExtendProps<
+  ComponentPropsWithRef<typeof Box>,
+  {
+    checked?: "indeterminate" | boolean;
+    children: ReactNode | string;
```

Comment:
Do we need to give `string` type? Can't we pass plain text with ReactNode?

Reply:
Unfortunately we can't. So we are wrapping it later with a `Text` component just in case if plain text is passed.